### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by ghost**
### What has been done
- Changed font
- Added shadow to the text
- Changed text color

### CODE REVIEW

**Review by mariusgrigaitis — CHANGES_REQUESTED** [review]
Font and shadow is ugly. I also think we changed font color for better contrast

**Comment by ghost** [comment]
This is why official brand used for :)

File: index.html [hunk 1]
```diff
@@ -23,6 +25,7 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/static/favicon-16x16.png" />
     <link rel="manifest" href="/static/site.manifest" />
     <link rel="mask-icon" href="/static/safari-pinned-tab.svg" color="#5bbad5" />
+    <link href="//fonts.googleapis.com/css?family=Paytone+One&amp;subset=latin-ext" rel="stylesheet">
```
Comment (muningis) [inline index.html:28]: Should be deferred .

File: index.html [hunk 2]
```diff
@@ -11,9 +11,11 @@
             display: flex;
             justify-content: center;
             align-items: center;
-            font-family: Helvetica, Arial, sans-serif;;
+            font-family: 'Paytone One', Helvetica, Arial, sans-serif;
             background: #ff6b00;
-            color: #000;
+            color: #fff;
```
Comment (muningis) [inline index.html:16]: This colour doesn't have good enough constrast ratio with background.

**Review by muningis — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (ghost) [reply index.html:16]: nfq

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html:28]: > By default, font requests are delayed until the render tree is constructed...

File: index.html (hunk 2 above)
Comment (muningis) [reply index.html:16]: Your point?

File: index.html (hunk 1 above)
Comment (muningis) [reply index.html:28]: This is css-load, not font load.

**Pull request closed without merging**